programs/merkle-distributor/src/instructions/new_distributor.rs

```diff
@@ -83,6 +86,8 @@ pub fn handle_new_distributor(
     clawback_start_ts: i64,
     enable_slot: u64,
     closable: bool,
+    total_bonus: u64,
```

Comment:
When `total_bonus` > 0, should we check `bonus_vesting_slot_duration` > 0 too to avoid mistakenly initialization and causes the `total_bonus` released immediately?

Reply:
I think no need
- We have verification script to verify all parameters again 
- Release immediately can be on purpose